### ISSUE TITLE
2.3.10.6. Calinski-Harabasz Index

### ISSUE THREAD

**Issue by Akai-ai** [apachecn/sklearn-doc-zh#437]
原文是：sklearn.metrics.calinski_harabasz_score，用法如下：
import numpy as np
from sklearn.cluster import KMeans
kmeans_model = KMeans(n_clusters=3, random_state=1).fit(X)
labels = kmeans_model.labels_
metrics.calinski_harabasz_score(X, labels)
中文版的这里，少了一个字母
![image](https://user-images.githubusercontent.com/80671006/181903971-375ba735-d559-4e96-abad-8b6604df2bbc.png)

**Comment by iTzAlver** [comment]
I am using 1.2.2 and it is okay:
mectircs.calinski_harabasz_score

**Comment by jiangzhonglian** [comment]
抱歉，现在才修正

<img width="720" alt="image" src="https://github.com/apachecn/sklearn-doc-zh/assets/9199175/92e74a32-fd09-4cea-93a7-b5bcc87ead09">
